tests/app/test_app.py

```diff
@@ -163,7 +163,7 @@ def test_missing_thread_id_does_not_cause_exception(self):
             self.client.post(url, data=json.dumps(test_message), headers=self.headers)
             self.assertTrue(True)  # i.e no exception
         except Exception as e:
-            self.fail("post raised unexpected exception: {0}".format(e))
+            self.fail(f"post raised unexpected exception: {e}")
```

Comment:
Could you log the exception using logger.exception, then just log the message text in self.fail using e.message?

Reply:
@AndrewTorrance I don't think so, as the test is for not raising a generic exception.

Reply:
Approving this based on the above test being refactored via a github issue.